src/features/farming/bakery/components/CraftingItems.tsx

```diff
@@ -72,12 +76,42 @@ export const CraftingItems: React.FC<Props> = ({ items }) => {
     shortcutItem(selected.name);
   };
 
+  const onCaptchaSolved = async (captcha: string | null) => {
+    await new Promise((res) => setTimeout(res, 1000));
+
+    gameService.send("SYNC", { captcha });
+
+    onClose();
+  };
+
+  const sync = () => {
+    gameService.send("SYNC", { captcha: "" });
+
+    onClose();
+  };
+
+  const restock = () => {
+    // setShowCaptcha(true);
```

Comment:
Note, we are skipping the captcha temporarily until Cloudflare issues are resolved.

We will uncomment this in the future 👍